### PR TITLE
Fixed #47

### LINKED ISSUE #47: loginDeployGate task doesn't open a browser
The task silently finishes without any error.

Seems like releated to https://github.com/gradle/gradle/issues/2826, an issue from Gradle 4.1.

### PR DESCRIPTION
**Pull request by henteko**
Fixed #47 

Using the `open/start/xdg-open/gnome-open` command to open a browser.

### CODE REVIEW

**Comment by henteko** [comment]
@tnj Please check this code!

File: src/main/groovy/com/deploygate/gradle/plugins/utils/BrowserUtils.groovy [hunk 1]
```diff
@@ -1,28 +1,70 @@
...
         }
         false
     }
 
+    static void openBrowserForMac(String url) {
+        "open $url".execute().waitFor()
+    }
+
+    static void openBrowserForWindows(String url) {
+        "cmd /c start $url".execute().waitFor()
```
Comment (tnj) [inline src/main/groovy/com/deploygate/gradle/plugins/utils/BrowserUtils.groovy]:
```
['cmd', '/c', 'start', url].execute().waitFor()
```

**Comment by henteko** [comment]
@tnj Thanks! Fixed issues!
Please check again!

**Comment by henteko** [comment]
Thanks!

**Pull request merged**